specification/values/indictment.md

```diff
@@ -0,0 +1,10 @@
+# Indictment Values
+
+Identifies if product/s listed are the origin of the fault or contain a dependency inherited through code of another source.
+
+## Values
+
+ - **Origin**:  The products identified are considered the origin of the vulnerability and contain the code or logic considered at fault.
+ - **Dependency**: The product identified are considered to contain the vulnerability due to code or logic inherited through a dependency to third party code
+ 
+ ![Indictment Graph](../figures/graphsnippets/ProductSnippet.png "Indictment Graph")
```

Comment:
Based on my previous comment, this set of values would not be needed.

Reply:
See latest updates to PR for myriad of changes made to support this different approach